py/bitbox02/bitbox02/bootloader.py

```diff
@@ -64,7 +65,7 @@ def __init__(self, transport: TransportLayer, device_info: DeviceInfo):
         self.expected_magic = {
             "bb02-bootloader": SIGDATA_MAGIC_STANDARD,
             "bb02btc-bootloader": SIGDATA_MAGIC_BTCONLY,
-            "bb02-base": SIGDATA_MAGIC_STANDARD,
+            "bb02-base": SIGDATA_MAGIC_BITBOXBASE_STANDARD,
```

Comment:
bb02-base is a weird product string, mixing two platforms. sounds like `bitboxbase-standard` is more fitting.

Reply:
`bitboxbase-bootloader` to match the other ones?

Reply:
Fixed.